src/nvim/os/shell.c

```diff
@@ -68,6 +68,18 @@ char **shell_build_argv(const char *cmd, const char *extra_args)
   return rv;
 }
 
+char **build_argv(char *cmd)
```

Comment:
Because that would call a shell inside a shell, which seemed wasteful, and confusing behaviour.